### DISCUSSION TITLE
GET /backend/generator/Fusio.Adapter.File.Generator.FileDirectory - No such file or directory

### DISCUSSION THREAD

**Discussion by blipur** [apioo/fusio#538] [Q&A]
![image](https://github.com/apioo/fusio/assets/62276883/9cc42263-b184-4da2-a6ed-a4086829df54)

![image](https://github.com/apioo/fusio/assets/62276883/19bd0301-ca33-484a-aeaa-d32b321ec1cc)

i get error in generator menu

**Comment by chriskapp** [comment]
Hi, if you use the File generator you normally need to provide a directory s.
![file_generator](https://github.com/apioo/fusio/assets/2505846/8a1cd5ac-2902-45b1-b4f0-e436e5716ede)

in your screenshot it looks like this input is not available. Fusio will then generate endpoints to work with those files.
The error basically say, that you have not provided the directory. These config fields below the class field depend on the
selected generator, so you could check whether those config fields are correctly loaded
